terminus-ui/form-field/src/form-field.component.ts

```diff
@@ -266,7 +266,7 @@ export class TsFormFieldComponent implements AfterContentInit, AfterContentCheck
     // Subscribe to changes in the child control state in order to update the form field UI.
     // NOTE: non-null-assertion needed to pass `null` to `startWith`
     // tslint:disable: no-non-null-assertion
-    this.control.stateChanges.pipe(startWith<void>(null!)).subscribe(() => {
+    this.control.stateChanges.pipe(startWith<void, string>(null!)).subscribe(() => {
```

Comment:
Are these related to stylelint or some other fix/enhancement got added? Does it have a ticket that needs to be closed?

Reply:
No other ticket and honestly not sure why I got hit by these in my branch. It's due to an RxJS update where they deprecated the version with a single type